lib/Runtime/Library/JSONStringifier.h

```diff
@@ -0,0 +1,107 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+
+namespace Js
+{
+
+struct JSONObjectStack
+{
+    RecyclableObject* object;
+    JSONObjectStack* next;
+
+    bool Has(_In_ RecyclableObject* object)
+    {
+        JSONObjectStack* stack = this;
+        while (stack != nullptr)
```

Comment:
Nit: Use list iterator macros?

Reply:
I don't have a real list, so not sure what macros you mean?

---
In reply to: [147308869](https://github.com/Microsoft/ChakraCore/pull/4077#discussion_r147308869) [](ancestors = 147308869)